base/Dockerfile

```diff
@@ -39,7 +39,8 @@ RUN if [ "${ST2_VERSION#*dev}" != "${ST2_VERSION}" ]; then \
   && echo ST2_REPO=${ST2_REPO} \
   && curl -sf https://packagecloud.io/install/repositories/StackStorm/${ST2_REPO}/script.deb.sh | bash \
   && apt-get install -y st2=${ST2_VERSION}-* \
-  && rm -f /etc/apt/sources.list.d/StackStorm_*.list
+  && rm -f /etc/apt/sources.list.d/StackStorm_*.list \
+  && apt-get install -y redis
```

Comment:
The redis client library should be installed in StackStorm virtualenv.
Additionally, it's ideal to include a small code comment in Dockerfile about why it's needed. 

```suggestion
  && rm -f /etc/apt/sources.list.d/StackStorm_*.list
  
# Install redis client library for st2 coordination backend
# See: https://docs.stackstorm.com/reference/ha.html#zookeeper-redis
# See: https://docs.stackstorm.com/latest/reference/policies.html
RUN /opt/stackstorm/st2/bin/pip install redis
```